app/subscriber/src/features/home/Home.tsx

```diff
@@ -27,18 +27,6 @@ export const Home: React.FC = () => {
   const [{ settings: appSettings }] = useLookup();
   const [content, setContent] = React.useState<IContentSearchResult[]>([]);
   const [selected, setSelected] = React.useState<IContentModel[]>([]);
-  const [isReady, setIsReady] = React.useState<boolean>(false);
```

Comment:
None of this seems like it's needed.  Anyone have input on what all this was supposed to do?